cocos/scene-graph/node-event-processor.ts

```diff
@@ -370,7 +370,7 @@ export class NodeEventProcessor {
         }
     }
 
-    public updateSiblingIndex () {
+    public onUpdatingSiblingIndex () {
```

Comment:
Will it break compatibility?

Reply:
no, NodeEventProcessor is a private class